helix-core/src/test/java/org/apache/helix/integration/task/TestDeleteJobFromJobQueue.java

```diff
@@ -69,7 +69,14 @@ public void testForceDeleteJobFromJobQueue() throws InterruptedException {
     Assert
         .assertNotNull(_driver.getJobContext(TaskUtil.getNamespacedJobName(jobQueueName, "job2")));
 
-    // The following force delete for the job should go through without getting an exception
+    // Force deletion can have Exception thrown as controller is writing to propertystore path too.
+    // https://github.com/apache/helix/issues/1406, also force deletion may not be safe.
+    // Thus, we stop pipeline to make sure there is not such race condition.
+    _gSetupTool.getClusterManagementTool().enableCluster(CLUSTER_NAME, false);
```

Comment:
CLUSTER_NAME is a defined in the parent class which is also used by other tests. How could you ensure disabling this CLUSTER_NAME would not impact or fail other tests that use the same cluster?

Reply:
`protected final String CLUSTER_NAME = CLUSTER_PREFIX + "_" + getShortClassName();` each test class has unique  cluster name.

Reply:
Ops, naming style as a constant but it is not... So easy to misread.